web/src/components/netflow-table-row.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+
+import { ParsedStream } from "../api/loki";
+import { Tr, Td } from "@patternfly/react-table";
+import { Columns } from "./netflow-table-header";
+import protocols from "protocol-numbers";
+
+import { ResourceLink } from "@openshift-console/dynamic-plugin-sdk";
+
+const NetflowTableRow: React.FC<{ flow: ParsedStream }> = ({
+  flow,
+}) => {
+  return (
+    <Tr>
+      <Td key={Columns.date}>
+        {new Date(flow.value.timestamp).toLocaleString()}
+      </Td>
+      <Td key={Columns.srcpod}>
+        {flow.value.IPFIX.SrcPod && (
+          <ResourceLink
+            kind="Pod"
+            name={flow.value.IPFIX.SrcPod}
+            namespace={flow.labels["SrcNamespace"]}
+          />
+        )}
+      </Td>
+      <Td key={Columns.dstpod}>
+        {flow.value.IPFIX.DstPod && (
+          <ResourceLink
+            kind="Pod"
+            name={flow.value.IPFIX.DstPod}
+            namespace={flow.labels["DstNamespace"]}
+          />
+        )}
+      </Td>
+      <Td key={Columns.srcnamespace}>
+        {flow.labels["SrcNamespace"] && (
+          <ResourceLink kind="Namespace" name={flow.labels["SrcNamespace"]} />
+        )}
+      </Td>
+      <Td key={Columns.dstnamespace}>
+        {flow.labels["DstNamespace"] && (
+          <ResourceLink kind="Namespace" name={flow.labels["DstNamespace"]} />
+        )}
+      </Td>
+      <Td key={Columns.srcport}>{flow.value.IPFIX.SrcPort}</Td>
+      <Td key={Columns.dstport}>{flow.value.IPFIX.DstPort}</Td>
+      <Td key={Columns.protocol}>{protocols[flow.value.IPFIX.Proto].name}</Td>
+      <Td key={Columns.bytes}>{flow.value.IPFIX.Bytes}</Td>
+      <Td key={Columns.packets}>{flow.value.IPFIX.Packets}</Td>
```

Comment:
We can use a for loop here + a switch case for each type as you did in `netflow-table.tsx `